plugin/pkg/shoot/seedmanager/admission.go

```diff
@@ -192,10 +209,6 @@ func determineSeed(shoot *garden.Shoot, seedLister gardenlisters.SeedLister, sho
 		}
 	}
 
-	if bestCandidate == nil {
```

Comment:
Why is this removed? I guess we need to keep it (to not `return nil, nil`), and maybe just improve the error message?

Reply:
We can add it at the end to be on the safe side. But a `nil, nil` return can never happen if we have at least 1 `candidate`. Otherwise we exit with an error before that.

Reply:
We added this PR because we ran into an issue where we had colliding CIDRs and the error message of the admission was extremely misleading.

Reply:
> We can add it at the end to be on the safe side. But a `nil, nil` return can never happen if we have at least 1 `candidate`. Otherwise we exit with an error before that.

I checked again, you're right, can never happen. So let's go back what you did before and completely remove that `if`-block. Thank you!

> We added this PR because we ran into an issue where we had colliding CIDRs and the error message of the admission was extremely misleading.

Sure, agreed! That's definitely an improvement - thank you!

Reply:
Alright. Will revert the stuff.